<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartManagerForm</class>
 <widget class="QWidget" name="PartManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1232</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="categoriesFrame">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget#widget {background-color:qlineargradient(spread:pad, x1:0.5, y1:0.8, x2:0.5, y2:0, stop:0.248619 rgba(67, 67, 67, 255), stop:0.707182 rgba(114, 114, 114, 255));}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="treeViewLabel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Categories</string>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2"/>
       </item>
       <item>
        <widget class="QTreeView" name="mainTreeView">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>true</bool>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="autoExpandDelay">
          <number>750</number>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="partsFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="filterSplitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:qlineargradient(spread:pad, x1:0.5, y1:0.8, x2:0.5, y2:0, stop:0.248619 rgba(67, 67, 67, 255), stop:0.707182 rgba(114, 114, 114, 255));</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Parts List</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="leftMargin">
              <number>6</number>
             </property>
             <item>
              <widget class="QToolButton" name="addPartToolButton">
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonFollowStyle</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="deletePartToolButton">
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonFollowStyle</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="moreButton">
               <property name="styleSheet">
                <string notr="true">QPushButton::menu-indicator {
    subcontrol-position: right center;
    subcontrol-origin: padding;
    left: -4px;
}</string>
               </property>
               <property name="text">
                <string>More</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_6" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <property name="spacing">
               <number>8</number>
              </property>
              <property name="leftMargin">
               <number>6</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>6</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="filterlabel1">
                <property name="text">
                 <string>Storage:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="filter1"/>
              </item>
              <item>
               <widget class="QLabel" name="filterlabel2">
                <property name="text">
                 <string>Category:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="filter2">
                <item>
                 <property name="text">
                  <string>All Categories</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>All Subcategories</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Selected Category</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="filterlabel3">
                <property name="text">
                 <string>Stock:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="filter3">
                <item>
                 <property name="text">
                  <string>Any</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Out of stock</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>In Stock</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Below Minimum</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="filterlabel6">
                <property name="text">
                 <string>Condition:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="filter6">
                <item>
                 <property name="text">
                  <string>New</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Used</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Damaged</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QSearchLineEdit" name="textSearchLineEdit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="moreButton_2">
                <property name="styleSheet">
                 <string notr="true">QPushButton::menu-indicator {
    subcontrol-position: right center;
    subcontrol-origin: padding;
    left: -4px;
}</string>
                </property>
                <property name="text">
                 <string>More</string>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="extraItemsLayout">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="leftMargin">
              <number>6</number>
             </property>
             <property name="rightMargin">
              <number>6</number>
             </property>
            </layout>
           </item>
           <item>
            <widget class="PartsTableView" name="partsTableView">
             <property name="styleSheet">
              <string notr="true">QPushButton::menu-indicator {    
    subcontrol-position: left center;
    subcontrol-origin: padding;
    left: -2px;
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="filterHeader" native="true">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:qlineargradient(spread:pad, x1:0.5, y1:0.8, x2:0.5, y2:0, stop:0.248619 rgba(67, 67, 67, 255), stop:0.707182 rgba(114, 114, 114, 255));</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <property name="leftMargin">
               <number>9</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Filter</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>139</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QToolButton" name="colapseFilterPanelButton">
                <property name="minimumSize">
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QToolButton {
	border:none;		
}

QToolButton:hover {
    border:none;
	background-color: rgba(160, 160, 160, 80);
}

QToolButton:pressed {
	border:none;
	background-color: rgba(50, 50, 50, 50);
}

</string>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonIconOnly</enum>
                </property>
                <property name="arrowType">
                 <enum>Qt::DownArrow</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="PartsFilterForm" name="filterForm" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="filterHeader_2" native="true">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:qlineargradient(spread:pad, x1:0.5, y1:0.8, x2:0.5, y2:0, stop:0.248619 rgba(67, 67, 67, 255), stop:0.707182 rgba(114, 114, 114, 255));</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Filter</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>139</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="expandFilterPanelButton">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton {
	border:none;		
}

QToolButton:hover {
    border:none;
	background-color: rgba(160, 160, 160, 80);
}

QToolButton:pressed {
	border:none;
	background-color: rgba(50, 50, 50, 50);
}

</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonIconOnly</enum>
            </property>
            <property name="arrowType">
             <enum>Qt::UpArrow</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="partDetailsframe">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:qlineargradient(spread:pad, x1:0.5, y1:0.8, x2:0.5, y2:0, stop:0.248619 rgba(67, 67, 67, 255), stop:0.707182 rgba(114, 114, 114, 255));</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Part Details</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="PartDetailsView" name="partDetailsView" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionAddCategory">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/folder_add</normaloff>:/icons/folder_add</iconset>
   </property>
   <property name="text">
    <string>Add Category</string>
   </property>
  </action>
  <action name="actionDeleteCategory">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/folder_delete</normaloff>:/icons/folder_delete</iconset>
   </property>
   <property name="text">
    <string>Delete Category</string>
   </property>
  </action>
  <action name="actionEditCategory">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/folder_edit</normaloff>:/icons/folder_edit</iconset>
   </property>
   <property name="text">
    <string>Edit Category</string>
   </property>
  </action>
  <action name="actionAddPart">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/plugin_add.png</normaloff>:/icons/images/plugin_add.png</iconset>
   </property>
   <property name="text">
    <string>Add Part</string>
   </property>
   <property name="toolTip">
    <string>Adds a new part to your database</string>
   </property>
  </action>
  <action name="actionDeletePart">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/plugin_delete.png</normaloff>:/icons/images/plugin_delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete Part</string>
   </property>
   <property name="toolTip">
    <string>Deletes the selected part</string>
   </property>
  </action>
  <action name="actionDuplicatePart">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/shape_square_add.png</normaloff>:/icons/images/shape_square_add.png</iconset>
   </property>
   <property name="text">
    <string>Duplicate</string>
   </property>
   <property name="toolTip">
    <string>Duplicate selected part</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSearchLineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/qsearchlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>PartsTableView</class>
   <extends>QTableView</extends>
   <header>widgets/partstableview.h</header>
  </customwidget>
  <customwidget>
   <class>PartDetailsView</class>
   <extends>QWidget</extends>
   <header>partdetailsview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PartsFilterForm</class>
   <extends>QWidget</extends>
   <header>partsfilterform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
